src/framework/theme/components/menu/menu-item.component.html

```diff
@@ -43,8 +43,11 @@
    href="#">
   <i class="menu-icon {{ menuItem.icon }}" *ngIf="menuItem.icon"></i>
   <span class="menu-title">{{ menuItem.title }}</span>
-  <i class="ion chevron" [class.ion-chevron-left]="!menuItem.expanded"
-     [class.ion-chevron-down]="menuItem.expanded"></i>
+  <i class="chevron"
+     [class.nb-arrow-right]="(isRtl$ | async) && !menuItem.expanded"
```

Comment:
Add [`BREAKING CHANGE:`](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md#footer) commit for this change